test/tz_stub.py

```diff
@@ -7,7 +7,7 @@
 import pytest
 
 try:
-    import zoneinfo  # type: ignore[import]
+    import zoneinfo
```

Comment:
Need to add this back, it complained on my machine that its an unused ignore, but fails in GitHub actions now that its removed.